src/apps/apps.tsx

```diff
@@ -82,6 +82,25 @@ export const apps: AppWithRouteConfig[] = [
     path: '/example-app-3',
     title: 'My Awesome App 3',
   },
+  {
+    name: 'JewelSwap',
+    component: lazy(() =>
+      import('./jewelswap/index').then((module) => ({
+        default: module.default,
+      })),
+    ),
+    link: 'jewelswap',
+    id: 'jewelswap',
+    description: 'Lend directly into JewelSwap',
+    imageUrlLight: ClaimEarningsLight,
+    imageUrlDark: ClaimEarningsDark,
+    isInstallable: true,
+    icon: <FileDownloadRoundedIcon />,
+    path: '/jewelswap',
+    title: 'Jewelswap',
+    transaction: sampleTransaction,
+    action: sampleAddAction,
```

Comment:
is it possible to have different light/dark images for your app? The icon as well (choose any available in the Material UI icons)